client/src/components/Q&A/Q&Alist.jsx

```diff
@@ -1,30 +1,28 @@
+/* eslint-disable react/prop-types */
 import React from 'react';
-import App from '../App.jsx';
-import axios from 'axios';
-import Question from './Question.jsx';
+import QuestionListEntry from './QuestionListEntry.jsx';
 
 const QuestionList = (props) => {
```

Comment:
You could consider destructuring your props (once you know what they'll be. You could also consider adding [type checking](https://reactjs.org/docs/typechecking-with-proptypes.html) to make sure that the correct data type is being passed in

Reply:
Good suggestions!